src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1507,5 +1514,44 @@ $$]]></Document>, extraExportedTypes:={GetType(CSharpEditorFormattingService)}.T
                 state.AssertMatchesTextStartingAtLine(6, "doo;")
             End Using
         End Sub
+
+        <WorkItem(588, "https://github.com/dotnet/roslyn/issues/588")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub DiffView()
+            Using state = TestState.CreateCSharpTestState(
+                <Document><![CDATA[
+class C
+{
+    void foo(int x)
+    {$$
+        /********/
+        int doodle;
+        }
+}]]></Document>, extraExportedTypes:={GetType(CSharpEditorFormattingService)}.ToList())
+
+                Dim differenceBufferService = state.GetExportedValue(Of IDifferenceBufferFactoryService)
+                Dim textBufferFactoryService = state.GetExportedValue(Of ITextBufferFactoryService)()
+                Dim contentTypeService = state.GetExportedValue(Of IContentTypeRegistryService)()
+                Dim contentType = contentTypeService.GetContentType(ContentTypeNames.CSharpContentType)
+                Dim textViewFactory = state.GetExportedValue(Of ITextEditorFactoryService)()
+                Dim editorOperationsFactory = state.GetExportedValue(Of IEditorOperationsFactoryService)()
+
+                Dim otherBuffer = textBufferFactoryService.CreateTextBuffer("text", contentType)
+                Dim otherExposedSpan = otherBuffer.CurrentSnapshot.CreateTrackingSpan(0, 4, SpanTrackingMode.EdgeExclusive, TrackingFidelityMode.Forward)
```

Comment:
A few comments would go a long way toward explaining what you're trying to pull off here.